src/content/1.1/code/agda/snippet04.agda

```diff
@@ -1,4 +1,10 @@
-f : A -> B
-g : B -> C
-h : C -> D
-h ∘ (g ∘ f) == (h ∘ g) ∘ f == h ∘ g ∘ f
\ No newline at end of file
+open import Function using (_∘_)
```

Comment:
![image](https://github.com/formalverification/milewski-ctfp-pdf/assets/10901543/ff6647f2-b573-4739-bcc6-7ca116d1e1d4)

Function composition does not render properly for me.